molecule/default/verify.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Verify role
+  hosts: all
+  become: yes
+
+  tasks:
```

Comment:
Ignore my previous comment: We **DEFINITELY** need a functional test here to verify that the role did its job. And to prevent human error by fools like me.

To test, I would manually find the sha1sum of one (or even all, if its not too much work) of the dummy config files, and then run a `stat` command on the path where the file should exist in its final destination, register the task, and then do assertions on the task's registered variable.

This will let you assert that the ownership, correct path, and sha1sum are as they should be for a given config file.

EG for the conf/7.x/solrconfig.xml file, it should wind up at /var/solr/data/test/conf/solrconfig.xml , be owned by `solr`/`solr`, and should have the same sha1sum as what you took manually.

Reply:
Done. @dale-c-anderson